e2e/automated.test.js

```diff
@@ -16,6 +16,11 @@ async function executeTest(url, targetUrl, selector) {
   await page.waitForSelector(`${selector}[href$="${targetUrl}"]`);
 }
 
+function createAnnotation({ file, lineNumber, ex }) {
+  // eslint-disable-next-line no-console
+  console.log(`::error file=${file},line=${lineNumber}::${ex.message}`);
```

Comment:
To support multi-line messages you need to encode `\n` with `%0A`

See https://github.com/mheap/phpunit-github-actions-printer/pull/14